virt/release_notes/virt-4-14-release-notes.adoc

```diff
@@ -179,25 +182,17 @@ Some features in this release are currently in Technology Preview. These experim
 
```

Comment:
Again, github won't let me comment on the relevant line... the "Notable technical changes" section heading should be commented out if it is going to be empty as well